src/PageContent/Calendar/DonatingAgencyContent.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import { AccountType } from '../../Enums';
+import './Content.css';
+import shelter from '../../icons/shelter.svg';
+
+class DonatingAgencyContent extends Component {
+    convertPhone(phone) {
+        if (phone !== undefined) {
+            let phoneString = phone.toString();
+            let formattedPhone =
+                phoneString.substr(0, 3) +
+                '-' +
+                phoneString.substr(3, 3) +
+                '-' +
+                phoneString.substr(6);
+            return formattedPhone;
+        }
+    }
+    render() {
+        let phone = this.convertPhone(this.props.donatingAgencyContactPhone);
+        return (
+            <div className="wrapper">
+                <img className="content-icon" src={shelter} alt="volunteer" />
+                <div className="content-wrapper">
+                    {this.props.accountType ===
+                    AccountType.DONATING_AGENCY_MEMBER ? (
+                            <h1 className="section-header">Recipient</h1>
```

Comment:
why would it show "Recipient" for donating agency member? 
This component should always only show the donating agency info, it should be that when you put together the dialog content for different account types, you choose to include and not include specific components based on the account type?